src/Framework/Framework/Resources/Scripts/serialization/date.ts

```diff
@@ -96,6 +104,41 @@ export function serializeDate(date: string | Date | null, convertToUtc: boolean
     return `${y}-${m}-${d}T${h}:${mi}:${s}.${ms}0000`;
 }
 
+export function serializeDateOnly(date: string | Date | null): string | null {
```

Comment:
As far as I can tell, it's only called with parameter of type Date, so wouldn't bother with validating the `string`